src/ux/requestsModal/ReviewRelayTransaction/ReviewTransactionContainer.tsx

```diff
@@ -203,16 +203,18 @@ export const ReviewTransactionContainer = ({
       Alert.alert(t('transaction_summary_insufficient_funds'))
     }
 
-    // get usd values
-    const tokenUsd = convertTokenToUSD(Number(value), tokenQuote)
```

Comment:
What was wrong with the convertTokenToUSD method? Does it be modified rather than reinventing the wheel locally in this file?